README.md

```diff
@@ -22,9 +22,15 @@ Memory Imaging for Linux Family by OpenLake scans through the virtual memory of
 Open index.html in Documentation/html to view the documentation.
   
 ### Documentation Instructions:
-1. install doxygen
-2. run Doxyfile with doxygen (type "doxygen Doxyfile" in the terminal while in the toothbrush directory) after completing your work
-3. commit the changes of Doumentation folder in the branch
+1. Install doxygen by running the following command:
+``` 
+sudo apt-get install doxygen
+```
+2. Run Doxyfile with doxygen (type "doxygen Doxyfile" in the terminal while in the toothbrush directory) after completing your work
+ ``` 
+ doxygen Doxyfile
+```
+3. Commit the changes of Doumentation folder in the branch
```

Comment:
Add a command here, following the pattern from above.
Otherwise it will feel out of place and lazy people will miss it.

Reply:
git checkout -b your-new-branch
git status
git add <files>
git commit -m <message>
git push origin your-new-branch
git log

are these commands correct?

Reply:
Just add commit and push. Also try to keep this a one line.

Reply:
git add .
git commit -a -m "first commit" 
git push

Reply:
> Just add commit and push. Also try to keep this a one line.
??